license.txt

```diff
@@ -0,0 +1,12 @@
+
+
+<!--
+Copyright (c) 2018 by freeCodeCamp (https://codepen.io/freeCodeCamp/pen/MJjpwO)
+
```

Comment:
- [ ] Maybe you should put your name here instead of FreeCodeCamp

Reply:
[OPTIONAL] This is optional but recommended I think the previous reviewer :point_up_2: missed this comment so please give more explanation and also give a good reason `why` you give these required changes :+1: